src/File.ts

```diff
@@ -24,6 +27,10 @@ export const initializeFile = ({
  * If you pass more than one replacement task, all subsequent tasks in the same
  * section must be re-rendered, as their section indexes change. Assuming that
  * this is done faster than user interaction in practice.
+ * 
+ * In addition, this function is meant to be called with reasonable confidence
+ * that the {@code originalTask} is unmodified and at the exact same line in the
```

Comment:
More splitting hairs, but I do think it's important the comments are accurate...

The following is from Task.ts:

```ts
    /** Line number where the section starts that contains this task. */
    public readonly sectionStart: number;
    /** The index of the nth task in its section. */
    public readonly sectionIndex: number;
```

The line number isn't exactly stored, it's the index of the task in its section. So I'm not sure that 'at the exact same line' is true...

Reply:
The line number is mentioned in the commit message as well.

Reply:
this should be fixed.